.github/workflows/copy-completed-notebooks.yml

```
@@ -0,0 +1,117 @@
+name: Copy completed HTML notebooks to training website repository
+# Copy completed HTML notebooks from the training-modules repo to this repo
+# This action will open a pull request containing completed HTML notebooks
+
+# This is a manually triggered workflow that takes two inputs:
+#  # The name of the workshop being taught
+#  # The training-modules repo tag to copy notebooks from
+on:
+  workflow_dispatch:
+    inputs:
+      training-module:
+        type: choice
+        description: Use the dropdown menu to select which training workshop module this website is being created for.
+        options:
+          - "Introduction to R and Tidyverse"
+          - "Introduction to single-cell RNA-seq"
+          - "Advanced single-cell RNA-seq"
+        required: true
+      training-modules-tag:
+        type: string
+        description: The `training-modules` repo tag to copy completed notebooks from, e.g. 2023-june
+        default: "master"
+        required: true
+
+jobs:
+  file-completed-notebooks-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout training-modules repository
+        uses: actions/checkout@v3
+        with:
+          repository: AlexsLemonade/training-modules
+          ref: ${{ github.event.inputs.training-modules-tag }}
+          path: training-modules
+
+      - name: Checkout this repository
+        uses: actions/checkout@v3
+        with:
+          path: main
+
+      - name: Configure git
+        run: |
+          # use global flag since we have multiple repos here
+          git config --global --add safe.directory "${GITHUB_WORKSPACE}"
+          git config --global user.email "actions@github.com"
+          git config --global user.name "GitHub Actions"
+
+      - name: Get HTML completed notebooks from training-modules
+        shell: bash
+        run: |
+          # The completed notebooks to copy will depend on which workshop module this is.
+          # For each module, the directory of notebooks to copy are:
+          #  Intro R: intro_to_R_tidyverse/
+          #  Intro scRNA-seq: intro_to_R_tidyverse/ and scRNA-seq/
+          #  Advanced scRNA-seq: advanced-scRNA-seq/
+
+          # the url for for this tag of the training-modules repo where files will be downloaded from
+          base_url=https://raw.githubusercontent.com/AlexsLemonade/training-modules/${{ github.event.inputs.training-modules-tag }}
+
+          ########### Get paths for files to copy, for the given module ############
+
+          # We have to cd into training-modules to ensure correct paths come out of `ls`
+          cd training-modules
+
+          # First, intro-to-R-tidyverse notebooks for the intro R workshop
+          if [[ "${{ github.event.inputs.training-module }}" == "Introduction to R and Tidyverse" ]]; then
+            notebook_files=`ls intro-to-R-tidyverse/*html`
+          fi
```

Can we simplify this to something like: 
```suggestion
          ########### Copy files for the given module ############
          
          # First, intro-to-R-tidyverse notebooks for the intro R workshop
          if [[ "${{ github.event.inputs.training-module }}" == "Introduction to R and Tidyverse" ]]; then
            cp training-modules/intro-to-R-tidyverse/*.html website/completed-notebooks/intro-to-R-tidyverse
          fi
```

---

Effectively accomplished but with `case`, not `if`, in https://github.com/AlexsLemonade/training-specific-template/pull/132/commits/1f922911a445ca2e9f15950249d7caa548ea24c5